### DISCUSSION TITLE
Error: Uncaught Error: [splide] null is invalid.

### DISCUSSION THREAD

**Discussion by BMUZAMMIL** [community/community#60181] [Programming Help]
### Select Topic Area

General

### Feature Area

Issues

### Body

Uncaught Error: [splide] null is invalid.
    bn assert.ts:12
    i Splide.ts:85
    <anonymous> (index):1064
    EventListener.handleEvent* (index):1063
assert.ts:12:10
*****************************************************
import { PROJECT_CODE } from '../../../constants/project';

/**
 * Throws an error if the provided condition is falsy.
 *
 * @param condition - If falsy, an error is thrown.
 * @param message   - Optional. A message to display.
 */
export function assert( condition: any, message?: string ): void {
  if ( ! condition ) {
    throw new Error( `[${ PROJECT_CODE }] ${ message || '' }` );
  }
}

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @BMUZAMMIL ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
